tree-sitter-darklang/grammar.js

```diff
@@ -310,6 +311,26 @@ module.exports = grammar({
         optional(alias(";", $.symbol)),
       ),
 
+    //
+    // Tuples
+    tuple_literal: $ =>
+      seq(
+        field("symbol_left_paren", alias("(", $.symbol)),
+        field("first", $.expression),
+        field("symbol_comma", alias(",", $.symbol)),
+        field("second", $.expression),
+        field("rest", optional($.rest_expressions)),
+        field("symbol_right_paren", alias(")", $.symbol)),
+      ),
+
+    rest_expressions: $ =>
```

Comment:
how about `tuple_literal_the_rest` or something? Each of these names should be really explicit